### DISCUSSION TITLE
关于Java运行环境bug问题

### DISCUSSION THREAD

**Discussion by LyingDoc** [1Panel-dev/1Panel#5978] [Q&A]
### 1Panel 版本

1.10.13

### 问题描述

首先这是我上面配置的Java附件上传位置，用这个java环境构建了，路径不会走这个，
这是我的启动命令： nohup java -Dname=pointlion-admin.jar  -Duser.timezone=Asia/Shanghai -Xms512m -Xmx1024m -XX:MetaspaceSize=128m -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -XX:+PrintGCDateStamps  -XX:+PrintGCDetails -XX:NewRatio=1 -XX:SurvivorRatio=30 -XX:+UseParallelGC -XX:+UseParallelOldGC -jar pointlion-admin.jar --spring.config.location=config/application.yml >>./logs.info &
下图是我上传附件之后的附件地址，你这个容器直接把上传路径修改了，
![图片](https://github.com/user-attachments/assets/9a92dde7-9d72-4a35-ad1d-954db273e497)

### 重现步骤

1，按照图片配置环境参数![图片](https://github.com/user-attachments/assets/581cfd64-2ccb-4ab9-89b1-4f3699459ab6)
2.直接启动，上传附件，例如上传头像
3，上传了之后我检查路径附件，r如图是空的，
![图片](https://github.com/user-attachments/assets/f659ec44-acaa-4224-80d0-39c34e00bcce)
bug：构建Java环境应用之后，容器上传路径被修改，常规来说直接识别运行目录路径，结果在乱码编号目录里面去了，

**Comment by zhengkunwang223** [comment]
没明白你的意思 
1. 你把 java 的代码放到了网站目录下 你这么做的目的是什么？
2. 你上传路径是文件的上传路径吗？ 容器内的路径和宿主机的路径是隔离的 所以 java 是操作的容器内路径吗？
